.oni/config.js

```diff
@@ -13,9 +13,11 @@ const deactivate = () => {
 module.exports = {
     activate,
     deactivate,
-    "workspace.testFileMappings": [{
-        sourceFolder: "browser/src",
-        mappedFolder: "browser/test",
-        mappedFileName: "${fileName}Tests.ts",
-    }]
+    "workspace.testFileMappings": [
```

Comment:
Need to sync up the formatting changes, there are a lot here that are unrelated to my change